packages/react-openapi/src/code-samples.ts

```diff
@@ -65,6 +65,24 @@ export const codeSampleGenerators: CodeSampleGenerator[] = [
             return lines.map((line, index) => (index > 0 ? indent(line, 2) : line)).join(separator);
         },
     },
+    {
+        id: 'python',
+        label: 'Python',
+        syntax: 'python',
+        generate: ({ method, url, headers, body }) => {
+            let code = 'import requests\n';
+            code += `response = requests.${method.toLowerCase()}("${url}",\n`;
+            if (headers) {
+                code += indent(`headers = ${JSON.stringify(headers, null, 2)},\n`, 11);
```

Comment:
The indentation looks weird in the screenshot; it looks like the code is not properly formatted.

Maybe we should try to make it look like https://codebeautify.org/python-formatter-beautifier

Reply:
Also can you look at enabling line wrapping for the code-block that we use for display?

Reply:
@SamyPesse Maybe I am doing it wrong, but adding line wrapping by modifying the css in the dev console makes the JS code look ugly
<img width="1127" alt="Screen Shot 2024-09-06 at 1 49 10 PM" src="https://github.com/user-attachments/assets/052be60c-18a7-497d-a531-1a67e37d76c1">

I am not good with CSS, so maybe I am missing something

Reply:
But if you think this looks fine, I will figure out where to implement it in code

Reply:
Here is what the new Python code looks like. Does this look okay to you? Tried to make it as close to Python prettifiers as possible
<img width="1115" alt="Screen Shot 2024-09-06 at 1 52 32 PM" src="https://github.com/user-attachments/assets/f8de40ba-6de9-4ba4-8f3b-e9fe1389bea0">

Reply:
Yeah looks good 👍 

I was not saying to do it with css for the wrapping, I think there is a prop for it (that is equivalent to the editor option)